common/src/main/java/com/linecorp/centraldogma/common/Change.java

```diff
@@ -65,10 +66,20 @@
     static Change<String> ofTextUpsert(String path, String text) {
         requireNonNull(text, "text");
         validateFilePath(path, "path");
-        if (EntryType.guessFromPath(path) == EntryType.JSON) {
```

Comment:
It seems not to make sense to use `ofTextUpsert` for not JSON but JSON5.
How about modifying `Change.ofJsonUpsert(String path, String jsonText)` to take JSON5?
```java
static Change<JsonNode> ofJsonUpsert(String path, String jsonText) {
  
    final JsonNode jsonNode = ...;
   
    return new DefaultChange<>(path, ChangeType.UPSERT_JSON, jsonNode, jsonText);
}
```
I think we can give `jsonText` a higher priority to serialize the data.